### ISSUE TITLE
Hardcoded path in error message

### ISSUE THREAD

**Issue by Dialga** [proofrock/ws4sqlite#12]
```
$ ./ws4sqlite -db db -serve-dir .
ws4sqlite 0.12.3
- Based on SQLite v3.39.4
panic: runtime error: slice bounds out of range [:-1]

goroutine 1 [running]:
main.parseCLI()
        /home/mano/ws4sqlite/src/cli.go:87 +0xdf9
main.main()
        /home/mano/ws4sqlite/src/ws4sqlite.go:67 +0x11b
```

**Comment by proofrock** [comment]
Thanks. Will fix.

**Comment by proofrock** [comment]
Should be fixed in the `develop` branch. If nothing more comes out, I will release a new version in a few days. Thanks!

**Comment by proofrock** [comment]
Released in v0.12.4
